### ISSUE TITLE
Stiction and Stribeck analisys

### ISSUE THREAD

**Issue by rbonghi** [robotology-playground/torque-control-params-estimation#5]
See on https://github.com/rbonghi/FrictionJoint/blob/master/scripts/test.m

![stribeck](https://cloud.githubusercontent.com/assets/4551246/6091914/f2418f16-aec3-11e4-885c-0eeb927d1fab.png)

If I fit data in only **positive acceleration** I recognize:
- Stiction effect
- Stribeck effect

I think is a good way to identify other properties of friction in data acquired @DanielePucci

**Comment by DanielePucci** [comment]
Yes, this is definitely interesting. What I am not sure about is the relative importance of the stribeck effects with respect to the global behavior of the friction. As a matter of fact, close to zero velocity, a model neglecting the stribeck would probably fail in approximating the above figure only for 1 Nm. I think, however, that it is worth investigating the modelling of this effect, and whether or not we have an improvements of the control performances with this additional feedforward term.

**Comment by rbonghi** [comment]
> What I am not sure about is the relative importance of the stribeck effects with respect to the global behavior of the friction.

It is not importance when you use this model for control, because IMHO the value of stribeck effect was "absorbed" on control compensation, but I think is more interesting on model estimation.

I would like to add a #4 to have a measure of good fitting data, because now I don't have an information about a precision of model fitting compare with data acquired.
